azurerm/resource_arm_signalr_service_test.go

```diff
@@ -398,6 +468,48 @@ resource "azurerm_signalr_service" "test" {
 `, rInt, location, rInt, capacity)
 }
 
+func testAccAzureRMSignalRService_withCors(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_signalr_service" "test" {
+  name                = "acctestSignalR-%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  cors {
+	allowed_origins = [
+	  "http://www.contoso.com",
+	  "www.contoso.com",
+	  "contoso.com",
+	]
```

Comment:
this is missing a closing `}` here:

```suggestion
	]
  }
```